docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -101,13 +101,13 @@ single number. For example, in case of age ranges, you can model the values as
   <<dfanalytics-types>>.
   
 `analyzed_fields`::
-  (Optional, object) You can specify both `includes` and/or `excludes` patterns. 
-  If `analyzed_fields` is not set, only the relevant fields will be included. 
-  For example, all the numeric fields for {oldetection}. For the supported field 
-  types, see <<ml-put-dfanalytics-supported-fields>>. If you specify fields – 
-  either in `includes` or in `excludes` – that have a data type that is not 
-  supported, an error occurs.
-  
+  (Optional, object) Specify `includes` and/or `excludes` patterns to select
```

Comment:
Probably that's outside of scope for this PR but I think the high-level analytics documentation should have an example in which it explains step by step the order of preprocessing actions i.e. `query` (can it use `excluded` fields?), `_source` and `analyzed_fields`.
@szabosteve: WDYT?

Reply:
@przemekwitek Good idea! There is an example section at the bottom of the page, I'll expand it after the PR merged. BTW, the regression and classification example pages contain something similar already (OFC without `_source`), putting a shorter example to the PUT page that focuses exclusively on this aspect is definitely beneficial.